components/director/pkg/graphql/schema.graphql

```diff
@@ -314,6 +314,7 @@ input ApplicationTemplateInput {
 	applicationInput: ApplicationRegisterInput!
 	placeholders: [PlaceholderDefinitionInput!]
 	accessLevel: ApplicationTemplateAccessLevel!
+	applicationNamespace: String
```

Comment:
Seems like the new `applicationNamespace` property is added only to the graphql `input` types. Does the use case also require us to expose it for reading? This would require adding it to `type ApplicationTemplate` as well.

Reply:
It is added in the type ApplicationTemplate as well. Maybe you missed it?

Reply:
ah, yes - my bad! 😄